purest-astropy failures in "background/tests/test_background_2d.py"

I just installed photutils and started to play with it.  I ran the pytest-astropy test after installation as recommend.

pytest --pyargs photutils

This generated three failures in the "test_background_2d.py" test.  All were of the form:

E       AssertionError: 
E       Arrays are not equal
E       
E       Mismatched elements: 2 / 16 (12.5%)
E       Max absolute difference among violations: 2.22044605e-16
E       Max relative difference among violations: 2.22044605e-16
E        ACTUAL: array([[1., 1., 1., 1.],
E              [1., 1., 1., 1.],
E              [1., 1., 1., 1.],
E              [1., 1., 1., 1.]])
E        DESIRED: array([[1., 1., 1., 1.],
E              [1., 1., 1., 1.],
E              [1., 1., 1., 1.],
E              [1., 1., 1., 1.]])

/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/numpy/_utils/__init__.py:85: AssertionE
______________________________________ TestBackground2D.test_coverage_mask[nan] _______________________________________


This was on macOS running on an Intel based Mac

Here is the complete output:

[Pytest_Failures.txt](https://github.com/user-attachments/files/18215797/Pytest_Failures.txt)

--- a/photutils/background/tests/test_background_2d.py
+++ b/photutils/background/tests/test_background_2d.py
@@ -238,8 +238,8 @@ class TestBackground2D:
             bkg2 = Background2D(data, (25, 25), filter_size=(1, 1), mask=mask,
                                 coverage_mask=mask, fill_value=0.0,
                                 bkg_estimator=MeanBackground())
-        assert_equal(bkg1.background_mesh, bkg2.background_mesh)
-        assert_equal(bkg1.background_rms_mesh, bkg2.background_rms_mesh)
+        assert_allclose(bkg1.background_mesh, bkg2.background_mesh)
+        assert_allclose(bkg1.background_rms_mesh, bkg2.background_rms_mesh)
 
     def test_mask_nonfinite(self):
         data = DATA.copy()
@@ -325,7 +325,7 @@ class TestBackground2D:
 
         data = np.ones((111, 121))
         bkg = Background2D(data, box_size=10, exclude_percentile=100)
-        assert_equal(bkg.background_mesh, np.ones((12, 13)))
+        assert_allclose(bkg.background_mesh, np.ones((12, 13)))
 
     def test_filter_threshold(self):
         """
@@ -489,7 +489,7 @@ class TestBackground2D:
         bkgimg2 = bkg2.background
         assert_equal(arr2, arr3)
 
-        assert_equal(bkgimg1, bkgimg2)
+        assert_allclose(bkgimg1, bkgimg2)
 
     @pytest.mark.parametrize('bkg_est', [MeanBackground(),
                                          SExtractorBackground()])

